metaflow/plugins/argo/argo_live_run_class.py

```diff
@@ -82,60 +86,68 @@ def trigger(
             time (in minutes) to wait for run to finish
         """
 
+        live_run = ArgoLiveRun(flow_name, alt_flow_id_info)
+
         if parameters is None:
             parameters = {}
 
         # convert dicts/lists to json-valid strings
         for key in parameters:
-            if isinstance(parameters[key], (dict, list)):  # TODO: discuss best way to filter
+            if isinstance(
+                parameters[key], (dict, list)
+            ):  # TODO: discuss best way to filter
                 try:
                     parameters[key] = json.dumps(parameters[key])
                 except:
-                    raise TypeError(f"Parameter with key '{key}' not supported.\n"
-                                    "Supported types are str and json-convertible objects")
+                    raise TypeError(
+                        f"Parameter with key '{key}' not supported.\n"
+                        "Supported types are str and json-convertible objects"
+                    )
 
-        print(f"template name: {self._template_name}")
+        print(f"template name: {live_run._template_name}")
 
         # trigger run and retrieve id info
-        flow_information = self._argo_client.trigger_workflow_template(
-            self._template_name,
+        flow_information = live_run._argo_client.trigger_workflow_template(
+            live_run._template_name,
             parameters=parameters,
         )
 
-        self._time_triggered = time.time()
-        self._argo_run_id = flow_information["metadata"]["name"]
-        self._metaflow_run_id = f"argo-{self._argo_run_id}"
+        live_run._time_triggered = time.time()
```

Comment:
minor: There should be a time from argo workflow if triggered successfully.

Reply:
ok as is